sdk/javascript/src/symbol/merkle.js

```diff
@@ -331,7 +352,7 @@ export const provePatriciaMerkle = (encodedKey, valueToTest, merklePath, stateHa
 		}
 
 		childHash = nodeHash;
-		actualPath = `${formattedLinkIndex}${uint8ToHex(node.path.path)}${actualPath}`;
+		actualPath = `${formattedLinkIndex}${node.hexPath}${actualPath}`;
```

Comment:
this is location of bug

in the case node path had odd nibble, original code would return full byte (resulting in extra 0 nibble in path).

correction is to trim it, which is now happening in hexPath

similar bug (and fix) applies to python sdk